tf2_ros/src/tf2_ros/buffer.py

```diff
@@ -67,6 +67,8 @@ def __init__(self, cache_time = None, debug = True):
             except (rosgraph.masterapi.Error, rosgraph.masterapi.Failure):   
                 self.frame_server = rospy.Service('~tf2_frames', FrameGraph, self.__get_frames)
 
+        self.CAN_TRANSFORM_POLLING_SCALE = 0.01
```

Comment:
This constant should be declared at the top of the file not in the middle of the function. It also doesn't need to be a member.